exodus/men_customization/templates/base/base.html

```diff
@@ -0,0 +1,123 @@
+{% load i18n %}
+<!doctype html>
+{% get_current_language as LANGUAGE_CODE %}
+<html lang="{{ LANGUAGE_CODE }}">
+  <head>
+    {% load static %}
+    <link rel="stylesheet" href="{% static "men_customization/css/bootstrap.men.css" %}">
+    <link rel="stylesheet" href="{% static "css/exodus.css" %}">
+    <link rel="shortcut icon" href="{% static "men_customization/point-amal.png" %}">
+
+    <script src="{% static "js/jquery.min.js" %}" ></script>
+    <script src="{% static "js/popper.min.js" %}" ></script>
+    <script src="{% static "js/bootstrap.min.js" %}" ></script>
+    <script src="{% static "js/handlebars.min.js" %}" ></script>
+
+    <style>
+<!--      .bg-primary  {-->
+<!--        background-color:rgb(0,94,139) !important;-->
+<!--      }-->
+
+<!--      .btn-primary {-->
+<!--        background-color:rgb(0,94,139) !important;-->
+<!--        border-color:rgb(0,94,139) !important;-->
+<!--      }-->
+
+<!--      .page-item.active .page-link {-->
+<!--        background-color:rgb(0,94,139) !important;-->
+<!--        border-color:rgb(0,94,139) !important;-->
+<!--      }-->
+
+      a.main-link {
+        color:rgb(0,94,139) !important;
+      }
+
+      .main-title {
+        color:rgb(0,94,139) !important;
+      }
+    </style>
```

Comment:
If we need extra CSS rules, I would suggest to make a men.css file and import it here

Reply:
Will be done in a separate CSS file.